src/VntuFaculties.java

```diff
@@ -0,0 +1,56 @@
+public enum VntuFaculties {
+    FCSA("Faculty of Computer Systems and Automation"),
+    FCHPGS("Faculty of Construction, Heat Power and Gas Supply"),
+    FPEE("Faculty of Power Engineering and Electromechanics"),
+    FITCE("Faculty of Information Technology and Computer Engineering"),
+    FMET("Faculty of Mechanical Engineering and Transport"),
+    FIRaN("Faculty of Infocommunications, Radioelectronics and Nanosystems"),
+    FMIS("Faculty of Management and Information Security"),
+    FESEM("Faculty of Ecological Safety and Environmental Monitoring");
+
+    private String facultyName;
+    private static VntuFaculties[] facultiesList = VntuFaculties.values();
```

Comment:
There is no needs to create this facultiesList, just use VntuFaculties.values() whenever you need it